api/resources_portal/migrations/0002_auto_20200604_2003.py

```diff
@@ -0,0 +1,28 @@
+# Generated by Django 2.2.12 on 2020-06-04 20:03
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("resources_portal", "0001_initial"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="materialrequest",
+            name="status",
+            field=models.CharField(
+                choices=[
+                    ("PENDING", "PENDING"),
+                    ("APPROVED", "APPROVED"),
+                    ("REJECTED", "REJECTED"),
+                    ("INVALID", "INVALID"),
+                    ("CANCELLED", "CANCELLED"),
+                ],
+                default="PENDING",
+                max_length=32,
+            ),
+        ),
+    ]
```

Comment:
Wait, can this one be rolled up into the initial migration? (Sorry if I've been confusing about this. Basically once we have a staging server up I want to start adding additional migrations, but let's start that after this PR since I gotta re-init the staging database anyway.